gulpfile.js

```diff
@@ -19,6 +18,7 @@ var sass = require('gulp-sass');
 var source = require('vinyl-source-stream');
 var streamify = require('gulp-streamify');
```

Comment:
`source` and `streamify` can be removed and uninstalled since they are no longer used.